configs/common/packages.yaml

```diff
@@ -40,7 +40,7 @@
     ecbuild:
       version: ['3.7.2']
     eccodes:
-      version: ['2.27.0']
+      version: ['2.32.0']
```

Comment:
the latest version is 2.33.0
I need to check is this 2.32.0 has the updated tables

Reply:
@climbfuji   if `NOAA` and `emc` need to bless the `2.33.0` we should hold off until we can update to this it contains the new WMO BUFR tables

there is also a `2.32.1` just fyi

Reply:
@BenjaminRuston Given that we are a solid week past the original code freeze date and that eccodes 2.32.0 is the latest available currently in spack develop, we'll go with this in spack-stack-1.6.0. We can update to 2.33.0 in 1.7.0 or in between.